src/Models/Environment.php

```diff
@@ -10,6 +10,7 @@
 use Pantheon\Terminus\Collections\Commits;
 use Pantheon\Terminus\Collections\Domains;
 use Pantheon\Terminus\Collections\Workflows;
+use Pantheon\Terminus\Helpers\LocalMachineHelper;
```

Comment:
Dang, the old Terminus class was set up to accept helpers and provide them through the application, but since we hadn't any I removed it.